staging/src/k8s.io/component-base/config/v1beta1/types.go

```diff
@@ -81,8 +81,7 @@ type ClientConnectionConfiguration struct {
 	Burst int32 `json:"burst"`
 }
 
-// LoggingConfiguration contains logging options
-// Refer [Logs Options](https://github.com/kubernetes/component-base/blob/master/logs/options.go) for more information.
+// LoggingConfiguration contains logging options.
 type LoggingConfiguration struct {
 	// Format Flag specifies the structure of log messages.
 	// default value of format is `text`
```

Comment:
I'm not sure what to do with the Sanitization field in the config while the feature is still experimental... it doesn't look like the feature is gated, and the config field doesn't indicate it is experimental like the command-line flag does, so it would be possible for someone to have this enabled in a config and not realize they are using an unstable feature (this is true today, I'm just wanting to use this promotion as a check of existing fields)

possibilities:
1. drop the field from the beta type
2. feature gate the functionality (failing if you try to set to set this to true without the feature enabled)
3. rename this to an explicitly experimental field

what do you think?